app/views/qsets/_qset_context.html.erb

```diff
@@ -1,32 +1,28 @@
 <span id="qset_context">
-  <h4>
-    Qset: <%= @qset.name %>
-    <% if can? :update, Qset %>
-      <a class="btn shortcut-button-link"
-         id="edit-qset"
-         data-toggle="modal"
-         data-target="#modal-edit-qset">
-        <span class="glyphicon glyphicon-edit shortcut-button-glyphicon"></span>
-      </a>
-      <!-- todo: delete button here? -->
+    <h3 class="center"><b><%= @qset.name %></b>
+    <% if @qset.parent.nil? == 'false' %>
```

Comment:
I'm not sure this would work... I think this should be 
```
<% if !@qset.parent.nil? %>
```
I tried it in a console:
```
irb(main):001:0> Qset.first
=> #<Qset id: 1, name: "Root", parent_id: nil>
irb(main):002:0> Qset.first.parent.nil?
=> true
irb(main):003:0> !Qset.first.parent.nil?
=> false
irb(main):004:0> !Qset.first.parent.nil? == 'false'
=> false
```